src/ext/ddtrace.c

```diff
@@ -89,6 +89,9 @@ static void ddtrace_shutdown(struct _zend_extension *extension) {
 static void ddtrace_activate(void) {}
 static void ddtrace_deactivate(void) {}
 
+// prepare the tracer state to start handling a new trace
+void _ddtrace_prepare_for_new_trace(TSRMLS_D);
```

Comment:
I suggest making it static since it's only used in this file, and we've been using the prefix `dd_` for such functions lately.

```suggestion
static void dd_prepare_for_new_trace(TSRMLS_D);
```